src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/JavaNCSSCheck.java

```diff
@@ -75,6 +75,15 @@
  * <pre>
  * &lt;module name="JavaNCSS"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public void test () {
+ *   System.out.println("Line 1");
+ *   ...
```

Comment:
please change to comment with explanation that there are another 49 lines

Reply:
Done